frontend/app/views/spree/taxons/show.html.erb

```diff
@@ -17,7 +17,7 @@
       </div>
     <% end %>
   </div>
-  <%= render 'subcategories', taxon: @taxon %>
+  <%= render 'subcategories', parent_taxon: @taxon.parent, taxon: @taxon %>
```

Comment:
Maybe instead of passing `@taxon.parent` through `parent_taxon`, we could access `@taxon.parent` inside the partial?